docs/en/user-manual/advanced_configuration.md

```diff
@@ -860,6 +860,29 @@ created on those organizations if `auto_create` is enabled.
 Refer to [Accessing Users in Other Organizations](https://adobe-apiplatform.github.io/user-sync.py/en/user-manual/advanced_configuration.html#accessing-users-in-other-organizations)
 for more information.
 
+## Two-Step Lookup
+
+Some LDAP systems may not support a `memberOf` overlay the way other systems like Active Directory do. This can make it
+impossible for the LDAP connector to use its group-based queries to get members for particular groups in an LDAP system.
+
+The `two_steps_lookup` option in `connector-ldap.yml` can be used to overcome this limitation. With two-step lookup
+enabled and configured, the sync tool will query all users (with respect to `all_users_filter`) and look for a specific
+user attribute to get group membership information. Users queried in this manner will be synced according to the `--users`
```

Comment:
@adorton-adobe
Not sure if the wording here is correct. Since we don't pull all the users first then filter.
Two steps lookup process:
1) Query Specified Group and retrieve group membership from specified "group_member_attribute_name" value. This will return only DN Value
2) Verify if the users are within the base dn
3) Lookup user dn by using all_users_filter filter to return Person Object.
4) Move on to the rest of connector ldap stuff.

The rest of the doc look good.